### PR TITLE
Increase Code Splitting

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR code splits more and removes some pointless packages overall reducing bundle size quite a bit.

We're under 1mb!

```
Compiled successfully.

File sizes after gzip:

  951.65 KB  dist/bundle.js
  373.23 KB  dist/103.bundle.js
  166.46 KB  dist/main.css
  43.94 KB   dist/0.bundle.js
  32.64 KB   dist/7.bundle.js
  12.09 KB   dist/10.bundle.js
  9.57 KB    dist/18.bundle.js
  9.15 KB    dist/17.bundle.js
  8.73 KB    dist/16.bundle.js
  5.76 KB    dist/1.bundle.js
  2.46 KB    dist/105.bundle.js
  2.02 KB    dist/13.bundle.js
  1.69 KB    dist/37.bundle.js
  1.68 KB    dist/38.bundle.js
  1.6 KB     dist/39.bundle.js
  1.58 KB    dist/35.bundle.js
  1.54 KB    dist/36.bundle.js
  1.5 KB     dist/14.bundle.js
  1.49 KB    dist/2.bundle.js
  1.33 KB    dist/9.bundle.js
  1.25 KB    dist/104.bundle.js
  1.23 KB    dist/11.bundle.js
  1.12 KB    dist/15.bundle.js
  885 B      dist/8.bundle.js
  844 B      dist/77.bundle.js
  840 B      dist/76.bundle.js
  816 B      dist/70.bundle.js
...
```

Blocked by https://github.com/opentripplanner/otp-ui/pull/476, https://github.com/opentripplanner/otp-ui/pull/481 and https://github.com/opentripplanner/otp-react-redux/pull/694

### CODE REVIEW

**Review by daniel-heppner-ibigroup — APPROVED** [review]
This seems to work very well for me and loads very quickly! Thank you!

File: lib/components/user/user-account-screen.js [hunk 1]
```diff
@@ -127,27 +124,29 @@ class UserAccountScreen extends Component {
             // can be wired to be managed by Formik.
             (formikProps) => {
               const DisplayComponent = isCreating
-                ? NewAccountWizard
-                : ExistingAccountDisplay
+                ? lazy(() => import('./new-account-wizard'))
+                : lazy(() => import('./existing-account-display'))
```
Comment (binh-dam-ibigroup) [inline lib/components/user/user-account-screen.js:128]: A deployment that has middleware persistence enabled will used either one, maybe it is `UserAccountScreen` that needs to be lazy-loaded from `ResponsiveWebApp`?

**Review by binh-dam-ibigroup — COMMENTED** [review]
Is the lazy loading of the user account content done in the correct spot? (see comment).

File: lib/components/user/user-account-screen.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/user/user-account-screen.js:128]: I tried doing lazy loading in `ResponsiveWebApp` and ran into problems with `<suspense>` placement. I think best practice is to split the app up into as many files as possible anyway, especially with `HTTP/3 and QUIC`

Comment (binh-dam-ibigroup) [reply lib/components/user/user-account-screen.js:128]: Are you/were you able to lazy load `UserAccountScreen` from `ResponsiveWebApp` based on config, without using `lazy`?

Comment (miles-grant-ibigroup) [reply lib/components/user/user-account-screen.js:128]: Oh sorry I misread your comment -- React components must be code-split using `lazy`

**Review by binh-dam-ibigroup — APPROVED** [review]
Nice work cleaning up and deduplicating libraries!

**Pull request merged**